front/components/featuredBrands/featuredBrands.html

```diff
@@ -1,10 +1,9 @@
-<div class="landing">
-    <div class="featured"
-      <carousel interval="10000">
-        <slide ng-repeat="slide in slides" active="slide.active">
-          <div text-align="centered">
-            <img ng-src="{{slide.image}}">
-          </div>
-        </slide>
-      </carousel>
-  </div>
\ No newline at end of file
+<div class="featured">
+  <carousel interval="10000">
+    <slide ng-repeat="slide in slides" active="slide.active">
+      <div text-align="centered">
```

Comment:
you should add a css for text-aligning center

you should not mix html with css